pkg/assume/entrypoint.go

```diff
@@ -67,8 +67,9 @@ func GetCliApp() *cli.App {
 				}
 				os.Exit(0)
 			}
-			if c.Bool("verbose") {
-				debug.CliVerbosity = debug.VerbosityDebug
+
+			if c.Bool("verbose") || os.Getenv("GRANTED_LOG") == "debug" {
+				debug.Enable()
```

Comment:
Once https://github.com/common-fate/clio/pull/4 is merged, we should be able to remove the `debug` package altogether from this repo.
```suggestion
			clio.SetLogLevelFromEnv("GRANTED_LOG")
			if c.Bool("verbose") {
				clio.SetLogLevelFromString("debug")
			}
```